### PR TITLE
fix(www): layout for `/guidelines/design-tokens` + convert to theme-ui

### PR DESCRIPTION
**Pull request by abhishekjakhar**
## Description

1. Converted the guidelines/design-tokens page to theme-ui
2. Fixed the desktop alignments & layout
3. ~Fixed the responsive layout(except tables)~
3. Fixed the responsive layout by making table horizontally scrollable

Local URL: http://localhost:8000/guidelines/design-tokens/
Production URL: https://www.gatsbyjs.org/guidelines/design-tokens/

#### Before(Actual)

font-sizes-actual

letter-spacing-actual

shado-elevation-actual

#### After(Fixed)

font-sizes-expected

letter-spacing-expected

shadow-elevation-expected

### CODE REVIEW

**Comment by abhishekjakhar** [comment]
@fk I have not updated the responsive for tables, I wanted to discuss the right approach with you, should we stack them column wise or make the overflow-x to scroll? 
I guess the responsive layout was like this before also, so I did not wanted to touch it before discussing with you.

**Comment by fk** [comment]
@abhishekjakhar Sorry for taking a bit longer for this one.
Regarding the table, I think … hmmm, yeah,

> overflow-x to scroll

like e.g. https://getbootstrap.com/docs/4.5/content/tables/#always-responsive is a good first step to take — not a huge effort, but a nice win for those who browse this on mobile.

What do you think?

**Comment by abhishekjakhar** [comment]
> @abhishekjakhar Sorry for taking a bit longer for this one.
> Regarding the table, I think … hmmm, yeah,
> 
> > overflow-x to scroll
> 
> like e.g. https://getbootstrap.com/docs/4.5/content/tables/#always-responsive is a good first step to take — not a huge effort, but a nice win for those who browse this on mobile.
> 
> What do you think?

@fk Yeah that looks good, in fact better than what we are having right now. 
I will update the PR to make the tables scroll horizontally. Thank you for sharing. 👍

**Comment by fk** [comment]
Thank _**you**_!! 😉❤️❤️

**Comment by abhishekjakhar** [comment]
> Thank _**you**_!! 😉❤️❤️

You're welcome 👍

**Comment by abhishekjakhar** [comment]
@fk I have updated the code to scroll tables horizontally on mobile layout by adding `overflowX: "auto"` property to the parent `div` of table element. I did this exactly like it was done on bootstrap link you shared with me.

File: www/src/pages/guidelines/design-tokens.js [hunk 1]
```diff
@@ -209,9 +241,13 @@ const DesignTokens = ({ location }) => (
             components along the z-axis.
           </p>
         </CopyColumn>
-        <ContentColumn fullWidth width="100%">
+        <ContentColumn overflowXMobile fullWidth>
```
Comment (abhishekjakhar) [inline www/src/pages/guidelines/design-tokens.js:244]: @fk I could have used `overflowX="auto"` and override the css in `<ContentColumn/>`, but we only needed this css rule on mobile(below md), so I had to explicitly add overflowXMobile props and check it inside `<ContentColumn/>` component.

Comment (fk) [reply www/src/pages/guidelines/design-tokens.js:244]: 👍

**Comment by abhishekjakhar** [comment]
> LGTM! 🙏 ❤️

Thank you very much 🙏

**Pull request merged**